applications/fishing-map/src/features/map/Map.tsx

```diff
@@ -251,10 +254,14 @@ const MapWrapper = (): React.ReactElement | null => {
           transformRequest={transformRequest}
           onResize={setMapBounds}
           getCursor={rulersEditing ? getRulersCursor : getCursor}
-          interactiveLayerIds={rulersEditing ? undefined : style?.metadata?.interactiveLayerIds}
+          interactiveLayerIds={
+            rulersEditing || drawMode !== 'disabled'
+              ? undefined
+              : style?.metadata?.interactiveLayerIds
+          }
           clickRadius={clickRadiusScale(viewport.zoom)}
-          onClick={currentClickCallback}
-          onHover={currentMapHoverCallback}
+          onClick={drawMode !== 'disabled' ? undefined : currentClickCallback}
```

Comment:
Same comment as in https://github.com/GlobalFishingWatch/frontend/pull/963/files#diff-f77b7b230099b5b21a1538c91c906e381bbaf58f88ed8aa3a4a4d696a483c46eR209